backends/tfhe-cuda-backend/cuda/src/utils/helper_multi_gpu.cu

```diff
@@ -11,31 +11,28 @@ int32_t cuda_setup_multi_gpu(int device_0_id) {
   int num_gpus = cuda_get_number_of_gpus();
   if (num_gpus == 0)
     PANIC("GPU error: the number of GPUs should be > 0.")
-  int num_used_gpus = 1;
```

Comment:
Maybe this is the cause of the failing test in the HL API? We should maybe change this in another PR?

Reply:
And rollback this, just in case.